src/gui/guiFormSpecMenu.cpp

```diff
@@ -1942,6 +1942,9 @@ void GUIFormSpecMenu::parseTabHeader(parserData* data, const std::string &elemen
 			pos = getRealCoordinateBasePos(v_pos);
 
 			geom = getRealCoordinateGeometry(v_geom);
+			// Set default height
+			if (parts.size() == 6)
```

Comment:
shouldn't this be `>= 6`?
honestly not sure because I have some difficulties understanding the code

Reply:
If parts is > 6 then the height will have been specified and we don't want to override it with the default.

Reply:
Ah, so the condition is actually equivalent to `parts.size() < 7`.